accelerator/mqtt_interface.c

```diff
@@ -0,0 +1,88 @@
+#include "config.h"
+#include "connectivity/mqtt/client_common.h"
+#include "connectivity/mqtt/duplex_callback.h"
+#include "connectivity/mqtt/duplex_utils.h"
+#include "errors.h"
+
+#define MQTT_INTERFACE_LOGGER "mqtt_interface"
+
+static logger_id_t logger_id;
+
+int main(int argc, char *argv[]) {
+  status_t ret;
+  mosq_config_t cfg;
+  struct mosquitto *mosq = NULL;
+
+  // Initialize logger
+  if (LOGGER_VERSION != logger_version()) {
+    return EXIT_FAILURE;
+  }
+
+  logger_init();
+  logger_color_prefix_enable();
+  logger_color_message_enable();
+  logger_output_register(stdout);
+  logger_output_level_set(stdout, LOGGER_DEBUG);
```

Comment:
These function are wrapped in `logger_helper_init`, you could use that instead.

Reply:
`logger_helper_init()`use  but `logger_output_level_set(stdout, LOGGER_WARNING);` not`logger_output_level_set(stdout, LOGGER_DEBUG);`. In my memory, it seems that the logging message won't be showed if we use `LOGGER_WARNING`.

Reply:
Since `logger_helper_init()` uses `logger_output_level_set(stdout, LOGGER_WARNING);` but not `logger_output_level_set(stdout, LOGGER_DEBUG);`, hence messages logged by  `log_debug`、 `log_info`、`log_notice` (whose levels are lower than `LOGGER_WARNING`) would not be shown in `stdout`.

Reply:
Since `logger_helper_init()` uses `logger_output_level_set(stdout, LOGGER_WARNING);` but not `logger_output_level_set(stdout, LOGGER_DEBUG);`, hence messages logged by `log_debug`、 `log_info`、`log_notice` (whose levels are lower than `LOGGER_WARNING`) would not be shown in `stdout`.

Reply:
And we can change the logger level with `logger_helper_enable` that we can reset the output level to `LOGGER_DEBUG`

Reply:
Sorry  I am kinda confused, so which solution should I take

Reply:
I wrote a explanation about logger usage [here](https://hackmd.io/@F7PvWMp5SY6blxnSo2iV6g/Byf-TMH7r).
Correct me if I misunderstand, thanks!

Reply:
I think the following would fit our needs:
```
logger_helper_init();
logger_output_level_set(stdout, LOGGER_DEBUG);
```

the `logger_helper_enable` does not reset the output level.
Correct me if I missed sth, @yiwei01

Reply:
Since we update entangled's version to **tag : "cclient-v1.0.0-beta"** in WORKSPACE, so we can pass argument to `logger_helper_init()` directly.
```
logger_helper_init(LOGGER_DEBUG);
```
, which directly sets stdout's level to LOGGER_DEBUG.